website/docs/pushdown.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 10
```

Comment:
Great job on the content! Since it's about performance optimization, I think we should have the position just below `operators`. @AlekSi can you advise on this?